theano/tensor/nnet/corr.py

```diff
@@ -377,8 +374,8 @@ class CorrMM(BaseCorrMM):
         Set to `(1, 1)` to disable subsampling.
 
     """
-    def __init__(self, border_mode="valid", subsample=(1, 1), openmp=None):
-        super(CorrMM, self).__init__(border_mode, subsample, openmp=openmp)
+    def __init__(self, border_mode="valid", subsample=(1, 1)):
```

Comment:
Sorry for the late response. That makes sense, will update the codes as you suggested.